components/net_traits/response.rs

```diff
@@ -318,30 +331,34 @@ impl Response {
                     match self.response_type {
                         ResponseType::Basic => Ok(FetchMetadata::Filtered {
                             filtered: FilteredMetadata::Basic(metadata.unwrap()),
-                            unsafe_: unsafe_metadata
+                            unsafe_: unsafe_metadata,
+                            net_timing: NetworkTiming::default(),
```

Comment:
Why do we create default values in various places here instead of using the actual values obtained earlier?